.github/workflows/latest-image.yml

```diff
@@ -0,0 +1,38 @@
+name: Docker Image - Development
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      -
+        name: Checkout 
+        uses: actions/checkout@v3
+      -
+        name: Login to Docker Hub
```

Comment:
Personally I never actually use DockerHub anymore. Because you are in Github I would recommend using ghcr sense it is easy. DockerHub isn't a wrong choice, the docker repo ecosystem is just a lot more diverse now that it was 5 years ago.

Reply:
Moved to ghcr.io